sklbench/datasets/loaders.py

```diff
@@ -792,11 +853,16 @@ def load_twodplanes(
 def load_year_prediction_msd(
     data_name: str, data_cache: str, raw_data_cache: str, dataset_params: Dict
 ) -> Tuple[Dict, Dict]:
-    url = (
-        "https://archive.ics.uci.edu/ml/machine-learning-databases/00203/"
-        "YearPredictionMSD.txt.zip"
+
+    ARCHIVE = (
+        (
+            "YearPredictionMSD.txt.zip",
+            "https://archive.ics.uci.edu/ml/machine-learning-databases/00203",
+            "06f801af323bb7798e800583acce4ea1ed2697ac12c23f4424aea0a7a3d09e11",
+        ),
     )

```

Comment:
```suggestion
    ARCHIVE = (
            "YearPredictionMSD.txt.zip",
            "https://archive.ics.uci.edu/ml/machine-learning-databases/00203",
            "06f801af323bb7798e800583acce4ea1ed2697ac12c23f4424aea0a7a3d09e11",
    )
```

This one is generating an error:
```
File "/localdisk2/mkl/mkltest/ec/Sklearnex-ML-Benchmarks-20251201232410/8957a2/sources/sklbench/datasets/loaders.py", line 865, in load_year_prediction_msd
 data = download_and_read_csv(ARCHIVE, raw_data_cache, header=None)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/localdisk2/mkl/mkltest/ec/Sklearnex-ML-Benchmarks-20251201232410/8957a2/sources/sklbench/datasets/downloaders.py", line 92, in download_and_read_csv
 logger.info(f"Downloading {file_data[0]} from {file_data[1]}...")
                                                ~~~~~~~~~^^^
IndexError: tuple index out of range
```